src/EditorFeatures/Core.Cocoa/Preview/PreviewFactoryService.cs

```diff
@@ -50,6 +50,20 @@ protected override async Task<ICocoaDifferenceViewer> CreateDifferenceViewAsync(
         {
             var diffViewer = _differenceViewerService.CreateDifferenceView(diffBuffer, previewRoleSet);
             diffViewer.ViewMode = mode;
+            const string DiffOverviewMarginName = "deltadifferenceViewerOverview";
+            if (mode == DifferenceViewMode.RightViewOnly)
+            {
+                diffViewer.RightHost.GetTextViewMargin(DiffOverviewMarginName).VisualElement.Hidden = true;
+            }
+            else if (mode == DifferenceViewMode.LeftViewOnly)
+            {
+                diffViewer.LeftHost.GetTextViewMargin(DiffOverviewMarginName).VisualElement.Hidden = true;
+            }
+            else
+            {
+                Contract.ThrowIfFalse(mode == DifferenceViewMode.Inline);
```

Comment:
Looks like you're missing a `using` for this. Should be `using Roslyn.Utilities;`